index.js

```diff
@@ -646,28 +1118,64 @@
     );
   };
 
-  var invalidReturnValue = function(name, types, value) {
+  //  badTypeVar :: (Number, String) -> Node
+  var badTypeVar = function(index, typeVarName) {
+    return isNaN(index) ? findRetTypeVar(typeVarName) :
+                          findArgTypeVar(index, typeVarName);
+  };
+
+  //  showVal :: (Integer, a, [Type]) -> String
+  var showVal = function(num, value, types) {
+    return num + ') ' + show(value) + ' :: ' + types.join(', ') + '\n\n';
```

Comment:
I'd like to have two spaces after the `)`. I'd like the value and the type to be more tightly bound visually than the value and the number.

One space:

    1) 1 :: Number

    2) "a" :: String

Two spaces:

    1)  1 :: Number

    2)  "a" :: String